distribution/ddf-common/src/main/resources/bin/ddf.bat

```diff
@@ -7,79 +7,47 @@ PUSHD %DIRNAME%..
 SET DDF_HOME=%CD%
 POPD
 
-SET GET_PROPERTY=%DIRNAME%get_property.bat
-SET SOLR_EXEC=%DDF_HOME%\solr\bin\solr.cmd
+IF "%COMMAND%"=="" (
```

Comment:
I don't think we want to prevent the user from starting ddf with just `ddf.bat`.

Reply:
Also... we never define COMMAND, so DDF cannot be started